tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py{35,36,37,38}-sphinx{20,21,22,23}
```

Comment:
If this tool happens to work on older versions of Python that is happenstance. For now, I would like to only support Python 3.8 (as per the README).

If you or someone else wishes to pledge maintainership for those older Pythons, that would change things. If any organisation wishes to pay for that maintainership, that also changes things.